data-protocols/dsp/dsp-lib/dsp-transfer-process-lib/dsp-transfer-process-transform-lib/src/main/java/org/eclipse/edc/protocol/dsp/transferprocess/transform/type/v2024/from/JsonObjectFromTransferProcessV2024Transformer.java

```diff
@@ -75,7 +75,8 @@ private String state(Integer state, String errorDetails, TransformerContext cont
         }
 
         return switch (transferProcessState) {
-            case INITIAL, REQUESTING, REQUESTED -> forNamespace(DSPACE_VALUE_TRANSFER_STATE_REQUESTED_TERM);
+            case INITIAL, REQUESTING, REQUESTED, PROVISIONED ->
```

Comment:
should `PROVISIONING` and `PROVISIONING_REQUESTED` also be included here?

Reply:
Done
https://github.com/eclipse-edc/Connector/pull/5129/commits/6bc005209df028edd8b2a4133925c80da3d358b3